VCFv4.4.draft.tex

```diff
@@ -322,9 +322,8 @@ \subsubsection{Fixed fields}
 
   \item ALT --- alternate base(s): Comma-separated list of alternate non-reference alleles.
   These alleles do not have to be called in any of the samples.
-  Options are base Strings made up of the bases A,C,G,T,N (case insensitive) or the `*' symbol (allele missing due to overlapping deletion) or a MISSING value `.' (no variant) or an angle-bracketed ID String (``$<$ID$>$'') or a breakend replacement string as described in Section \ref{Breakends}.
+  Options are base Strings made up of the bases A,C,G,T,N (case insensitive, non-zero length) or the `*' symbol (allele missing due to overlapping deletion) or a MISSING value `.' (no variant) or an angle-bracketed ID String (``$<$ID$>$'') or a breakend replacement string as described in Section \ref{Breakends}.
   If there are no alternative alleles, then the MISSING value must be used.
-  In other words, the ALT field must be a symbolic allele, or a breakend replacement string, or match the regular expression \texttt{\^{}([ACGTNacgtn]+|\string\*|\string\.)\$}.
```

Comment:
I'm guessing that this is incorrect because the sentence states that the ALT field follow this pattern where it is the alleles that does and the ALT field is a coma separated list of allele.
Wouldn't it better to change the `ALT` -> `allele` ? 

In any case this definition was also change in 4.3 and should be updated there as well.